src/utils.rs

```diff
@@ -74,6 +74,10 @@ Elixir 1.10 (compiled with Erlang/OTP 22)\n",
             stdout: String::from("v3.1.1+gafe7058\n"),
             stderr: String::default(),
         }),
+        "java -version" => Some(CommandOutput {
```

Comment:
I think this might need to be changed since the actual command that is run depends on if the  `JAVA_HOME` env var is set or not. So this command could either be `${JAVA_HOME}/bin/java -Xinternalversion` or `java -Xinternalversion`.

Maybe something like 
```suggestion
        s if s.ends_with("java -Xinternalversion" => Some(CommandOutput {
```

Reply:
Oh, this is really silly mistake from my end 🤦 
Thanks for catching it 👍